themes/rad/layouts/index.html

```diff
@@ -4,25 +4,25 @@
     <div id="peer">
       <h2>Peer–to–peer</h2>
       <p>
-        The Radicle stack builds on IPFS, which means it works without a centralized server. No hassle with setting up
-        your own hosting, and no need to trust companies with your data.
+        Open source development is a peer&ndash;to&ndash;peer process.
+        With Radicle you can collaborate directly, without giving your
+        data to third parties or setting up your own server.
       </p>
     </div>
     <div id="term">
       <h2>Terminal-first</h2>
       <p>
-        A Radicle project contains a git repository as well as the
-        issues and proposals associated with that repo. All accessible from
-        your terminal.
+        A Radicle project contains a git repository, plus the associated
+        issues and proposals. Access all the conversations
+        around your codebase from the comfort of your command line.
```

Comment:
I like this one.

Reply:
I would drop the "at the comfort of". Sounds a bit too startup-y.